plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/meta/CubridMetaModel.java

```diff
@@ -53,100 +65,362 @@ public String getTableOrViewName(GenericTableBase base)
+        String name = JDBCUtils.safeGetString(dbResult, "name");
+        String description = JDBCUtils.safeGetString(dbResult, "comment");
+        String tableName = JDBCUtils.safeGetString(dbResult, "target_class_name");
+        String owner = JDBCUtils.safeGetString(dbResult, "target_owner_name");
+        DBRProgressMonitor monitor = dbResult.getSession().getProgressMonitor();
+        CubridTable cubridTable = (CubridTable) container.getDataSource().findTable(monitor, null, owner, tableName);
+        return new CubridTrigger(cubridTable, name, description, dbResult);
+    }
+
+    @Override
+    public void loadProcedures(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull GenericObjectContainer container)
+            throws DBException {
+        try (JDBCSession session = DBUtils.openMetaSession(monitor, container, "Load procedures")) {
+            String sql = "select * from db_stored_procedure where owner = ?";
+            try (JDBCPreparedStatement dbStat = session.prepareStatement(sql)) {
+                dbStat.setString(1, container.getName());
+                try (JDBCResultSet dbResult = dbStat.executeQuery()) {
+                    while (dbResult.next()) {
+                        String procedureName = JDBCUtils.safeGetString(dbResult, "sp_name");
+                        String description = JDBCUtils.safeGetString(dbResult, "comment");
+                        String type = JDBCUtils.safeGetString(dbResult, "sp_type");
+                        String target = JDBCUtils.safeGetString(dbResult, "target");
+                        String returnType = JDBCUtils.safeGetString(dbResult, "return_type");
+                        DBSProcedureType procedureType;
+                        if (type.equalsIgnoreCase(CubridConstants.TERM_PROCEDURE)) {
+                            procedureType = DBSProcedureType.PROCEDURE;
+                        } else if (type.equalsIgnoreCase(CubridConstants.TERM_FUNCTION)) {
+                            procedureType = DBSProcedureType.FUNCTION;
+                        } else {
+                            procedureType = DBSProcedureType.UNKNOWN;
+                        }
+                        container.addProcedure(new CubridProcedure(container, procedureName, description, procedureType, target, returnType));
+                    }
+                }
+            }
+        } catch (SQLException e) {
+            throw new DBException("Load procedures failed", e);
+        }
+    }
+
+    @Nullable
+    @Override
+    public String getViewDDL(@NotNull DBRProgressMonitor monitor, @Nullable GenericView object, @Nullable Map<String, Object> options) throws DBException {
+        JDBCSession session = DBUtils.openMetaSession(monitor, object, "load view ddl " + object.getName());
+        String ddl = "-- View definition not available";
+        String regex = "\\[|\\]";
+        try {
+            String sql= String.format("show create view %s", object.getName());

```

Comment:
Please use statements parameters here

Reply:
I have tried using statements parameters here but it doesn't work with this query.

Reply:
Can you use a fully qualified name here? Or are views also without parents?

Reply:
Cubrid views support parents like Cubrid Table. Now, I have changed this to method getUniqueName().